code/modules/hydroponics/seeds.dm

```diff
@@ -59,6 +59,8 @@
 	var/grafted = FALSE
 	///Type-path of trait to be applied when grafting a plant.
 	var/graft_gene
+	///Determines if the plant should be allowed to mutate early at 30+ instability.
+	var/mutate_early = TRUE
```

Comment:
Could you add booleans to a bitflag variable? Something like `seed_flags = MUTATE_EARLY`